Sources/App/Features/DropIns/Controllers/DropInRouteController.swift

```diff
@@ -55,3 +55,61 @@ struct DropInRouteController: RouteCollection {
         return try await request.view.render("Admin/dropins-print", context)
     }
 }
+
+struct DropInSessionViewModel: Codable {
```

Comment:
This is just moving models from old world to the print page, this'll be rewritten at some point soon - and the whole file will be deleted then. So for now am happy to keep here